Taarafo.Core.Tests.Unit/Services/Foundations/Comments/CommentServiceTests.Logic.RetrieveById.cs

```diff
@@ -0,0 +1,48 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using FluentAssertions;
+using Force.DeepCloner;
+using Moq;
+using System;
+using System.Threading.Tasks;
+using Taarafo.Core.Models.Comments;
+using Xunit;
+
+namespace Taarafo.Core.Tests.Unit.Services.Foundations.Comments
+{
+    public partial class CommentServiceTests
+    {
+        [Fact]
+        public async Task ShouldRetrieveCommentByIdAsync()
+        {
+            // given
+            Guid randomCommentId = Guid.NewGuid();
+            Guid inputCommentId = randomCommentId;
```

Comment:
inputCommentId should be from the randomComment.

```csharp
Guid inputCommentId = randomComment.Id

Reply:
@ShriHumrudha have updated this